examples/getting-started-app/stop.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 
-if [ -e ./dev/pids ]
-then
-    cat ./dev/pids | xargs kill -SIGINT || true
-    rm ./dev/pids
+if [[ "$1" ]]; then
```

Comment:
It's redundant because we don't have path with length more than 100 chars in CI.